python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## 1.4.0
+
+### Feature:
+- Support for SFTP as file location [#1481](https://github.com/astronomer/astro-sdk/pull/1481)
+- Support for FTP as file location [#1482](https://github.com/astronomer/astro-sdk/pull/1482)
+- Add support for Azure Blob Storage [#1275](https://github.com/astronomer/astro-sdk/pull/1275)
+- Add source code facet for operators [#1537](https://github.com/astronomer/astro-sdk/pull/1537)
+- Use load_options param to pass pandasOptions in load_file [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
+- Add a snowflake load options [#1516](https://github.com/astronomer/astro-sdk/pull/1516)
+- Expose get_file_list_func to users [#1380](https://github.com/astronomer/astro-sdk/pull/1380)
+
+### Improvements
+- Generate constraints on releases and pushes (not PRs) [#1472](https://github.com/astronomer/astro-sdk/pull/1472)
+- Refactor snowflake merge function for easier maintenance [#1493](https://github.com/astronomer/astro-sdk/pull/1493)
+- Deprecate export_table_to_file to export_to_file [#1503](https://github.com/astronomer/astro-sdk/pull/1503)
+- Fix the dataframe test function name [#1490](https://github.com/astronomer/astro-sdk/pull/1490)
+- Decouple unit and integration tests for python-sdk [#1487](https://github.com/astronomer/astro-sdk/pull/1487)
+- Use autospec=True for mocks [#1500](https://github.com/astronomer/astro-sdk/pull/1500)
+
+### Bug fixes
+- Loadfile should also check for conn_type of conn_id provided [#1471](https://github.com/astronomer/astro-sdk/issues/1471)
+- Fix pre-commit issues [#1536](https://github.com/astronomer/astro-sdk/pull/1536)
```

Comment:
Is this change meaningful for end-users?

Reply:
same as https://github.com/astronomer/astro-sdk/pull/1543#discussion_r1066774377